pubtools/_quay/iib_operations.py

```diff
@@ -136,6 +137,7 @@ def _remove_index_image_signatures(
         current_signatures (list): List of current signatures.
         target_settings (dict): Target settings.
     """
+    set_aws_kms_environment_variables(target_settings, "cosign_signer")
```

Comment:
How will AWS key be used in removing signatures?

Reply:
yeah, it probably doesn't make sense to put it there as well